src/modules/client/ServerHTML.js

```diff
@@ -36,6 +36,10 @@ const ServerHTML = ({ locale, title, description, body, image, permalink }: Prop
 			<title>{title}</title>
 
 			<link rel='image_src' href={image} />
+
+			<link rel='stylesheet' type='text/css' href=
```

Comment:
Including it here means the stylesheet will be served by every route that doesn't need it.